frontend/src/components/viewers/Tensorboard.tsx

```diff
@@ -269,6 +306,7 @@ class TensorboardViewer extends Viewer<TensorboardViewerProps, TensorboardViewer
         deleteDialogOpen: false,
         podAddress: '',
         tensorflowVersion: DEFAULT_TENSORBOARD_VERSION,
+        tensorboardReady: false,
```

Comment:
nit: maybe also reset tensorboardReady to false when we start a tensorboard

Reply:
added in the _startTensorboard